edxval/migrations/0002_data__default_profiles.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+DEFAULT_PROFILES = [
+    "desktop_mp4",
+    "desktop_webm",
+    "mobile_high",
+    "mobile_low",
+    "youtube",
+]
+
+
+def create_default_profiles(apps, schema_editor):
+    Profile = apps.get_model("edxval", "Profile")
+    for profile in DEFAULT_PROFILES:
+        Profile.objects.get_or_create(profile_name=profile)
+
+
+def delete_default_profiles(apps, schema_editor):
+    Profile = apps.get_model("edxval", "Profile")
+    for profile in DEFAULT_PROFILES:
+        obj, _ = Profile.objects.get_or_create(profile_name=profile)
```

Comment:
Why get_or_create ? We need to delete them all. `Profile.objects.filter(profile_name=profile).delete()`